OsmAnd/src/net/osmand/plus/AppInitializer.java

```diff
@@ -337,6 +337,7 @@ public void onCreateApplication() {
 		app.model3dHelper = startupInit(new Model3dHelper(app), Model3dHelper.class);
 		app.trackSortModesHelper = startupInit(new TrackSortModesHelper(app), TrackSortModesHelper.class);
 		app.explorePlacesProvider = startupInit(new ExplorePlacesOnlineProvider(app), ExplorePlacesOnlineProvider.class);
+		app.coordinatesGridHelper = startupInit(new CoordinatesGridHelper(app), CoordinatesGridHelper.class);
```

Comment:
Probably better store it inside OsmandMapTileView, we have to many helpers in application class